python/tk_multi_launchapp/software_entity_launcher.py

```diff
@@ -252,11 +259,11 @@ def _get_sg_software_entities(self):
             # OR User restrictions.
             sw_filters.append(
                 {
-                    "filter_operator": "or",
+                    "filter_operator": "any",
```

Comment:
Although `or` is valid, it is not documented and mockgun does not recognize it, so switching to `any` which will have the same behaviour.